README.rst

```diff
@@ -124,16 +124,20 @@ Depending on your OS and environment, some of these packages may be more challen
 or any issue regarding the above packages, please check their respective documentation.
 
 
-Having trouble with PySCF?
+Quantum Chemistry Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Tangelo can be used without having a quantum chemistry package installed but many algorithms, by default, depend on one being installed.
 
-Although PySCF is installed by default, this behaviour can be turned off by defining the environment variable `NO_PYSCF=1` and then
-installing using pip by `pip install git+https://github.com/goodchemistryco/Tangelo.git@main` or using the "From source, using setuptools"
-method. 
+The two quantum chemistry packages that are natively supported are `PySCF <https://pyscf.org/>`_ and `Psi4 <https://psicode.org/>`_.
 
-The classical solvers `CCSDSolver`, `MP2Solver`, `FCISolver` and `MINDO3Solver` as well as `DMETProblemDecomposition` require 
-PySCF at this time.
+`DMETProblemDecomposition` is only supported with `PySCF`.
+`SemiEmpircalSolver` requires PySCF  along with 'python3 -m pip install git+https://github.com/pyscf/semiempirical'.
 
-Psi4 is supported everywhere except `CCSDSolver`, `MP2Solver`, `FCISolver` and `MINDO3Solver` and `DMETProblemDecomposition`.
+You are also welcome to provide your own interface to a quantum chemistry package of your choice by defining a subclass of
```

Comment:
We have a test that illustrates that: instead of trying to explain it, a link to the test I think would give much more. Let's be as concise as possible and reuse our resources rather than "duplicate" or "reformulate" what already exists.

A good question is whether or not we should inject / illustrate the feature in a notebook or not (maybe not worth the hassle). We don't have to mention a notebook or get a notebook ready for this PR. Let's just think about whether it's desirable, write it down here, and resolve this comment.

Reply:
I think there should be a notebook to some other package but the link, to the test, I added  should be good enough for now.